scAR/integration_tests/scar_integration_test.py

```diff
@@ -0,0 +1,26 @@
+from scAR import model
```

Comment:
Good question. 

Usually developers are in charge of running the unit tests locally to ensure they didn't break the code, by turning this integration test into a unit test this means it will be executed locally and therefore the developers have to ensure that all the requirements have been installed/set up on their local machine. 
I am not saying we shouldn't, we could I guess, but maybe keeping it separate is clearer? Any thought @Tobias-Ternent?

Reply:
How long does this current integration test take to execute? If it's pretty quick then I don't see any real harm in this becoming a unit test instead, but if it's going to take too long then it's not worthwhile and it's better to keep it separate. Looks like this would also cover a fair chunk of code too with a subset of 'real' data, so I don't really see any problems with that.

Reply:
10-15 seconds, so I guess I will convert it to a unit test.